todo_app/app.py

```diff
@@ -1,15 +1,21 @@
-from flask import Flask
-
+from flask import Flask, render_template, session, request, redirect
 from todo_app.flask_config import Config
+from todo_app.data.session_items import _DEFAULT_ITEMS, get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    # return 'Hello World!'
```

Comment:
Let's remove old code rather than comment it!

Reply:
Will do.